frame/lottery/src/benchmarking.rs

```diff
@@ -55,12 +58,12 @@ benchmarks! {
 		let set_code_index: CallIndex = Lottery::<T>::call_to_index(
 			&frame_system::Call::<T>::set_code{ code: vec![] }.into()
 		)?;
-		let already_called: (u32, Vec<CallIndex>) = (
+		let already_called: (u32, BoundedVec<CallIndex, T::MaxCalls>) = (
 			LotteryIndex::<T>::get(),
-			vec![
+			BoundedVec::<CallIndex, T::MaxCalls>::try_from(vec![
 				set_code_index;
 				T::MaxCalls::get().saturating_sub(1) as usize
-			],
+			]).unwrap(),
```

Comment:
I recently added a `bounded_vec!` macro for very basic usages. I think you can easily make it understand `vec![_; _]` syntax as well and then use it here.

Reply:
I can't find the macro in the code-base, can you point me to it please?

Reply:
ahh it is in https://github.com/paritytech/substrate/pull/10601, maybe you can review it ? :p

Reply:
not a big deal, we can proceed without it. 

Would be good if either of us remembers to come clean it up after the aforementioned PR is merged.

Reply:
Ok thanks.